product_supplierinfo_qty_multiplier/models/purchase_order_line.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2015 FactorLibre - Hugo Santos
+# Copyright 2021 Tecnativa - Víctor Martínez
+# Copyright 2021 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from math import ceil
+
+from odoo import _, api, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    @api.onchange("product_qty")
+    def _onchange_quantity(self):
+        super()._onchange_quantity()
+        if not self.product_id:
+            return
+        params = {"order_id": self.order_id}
+        seller = self.product_id._select_seller(
+            partner_id=self.partner_id,
+            quantity=self.product_qty,
+            date=self.order_id.date_order and self.order_id.date_order.date(),
+            uom_id=self.product_uom,
+            params=params,
+        )
+        qty = self.product_qty
+        if seller and seller.multiplier_qty and qty % seller.multiplier_qty:
```

Comment:
You can combine these two I think. @pedrobaeza correct me if I'm wrong...
```suggestion
        if seller.multiplier_qty and qty % seller.multiplier_qty:
```

Reply:
`seller` can be `False`, not an empty recordset, so it's correct as it is.